### PR TITLE
Fix bug: chinese can not input correctly

### PR DESCRIPTION
**Pull request by yangxiuwu**
When using the Chinese input method to input text in text attribute, you cannot input Chinese correctly in the text area. The text area will treat the pinyin letters during the input process as the input text.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by yangxiuwu** [comment]
Hi @bsekachev, I’m very sorry that it took so long to reply to you due to the National Day holiday.
The input effect before fix is as shown in the video below
https://github.com/opencv/cvat/assets/17425009/a312627c-1af3-4451-a129-b11b8c81b599
The input effect after fix is as shown in the video below
https://github.com/opencv/cvat/assets/17425009/32566731-8a7c-4065-9819-1ff8c6f484dc
The reasons are as follows: Chinese must be input through Chinese input software (such as [Sogou pinyin](https://pinyin.sogou.com/mac/) and cannot be input directly. The OnChange event will be triggered every time when a Pinyin letter is entered while inputting pinyin.After user finish input pinyin, the input software will convert the pinyin letter to Chinese. The pinyin letters entered during the input process are saved through temporary variables in useState and get the final input result of Pinyin through useEffect

**Comment by nmanovic** [comment]
@yangxiuwu , could you please add a changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/opencv/cvat/pull/6918>)
 
 ### Fixed
+- Can not input Chinese correctly in text attribute (<https://github.com/opencv/cvat/pull/6916>)
```
Comment (nmanovic) [inline CHANGELOG.md:59]: @yangxiuwu , you don't need to modify the file directly. Please look at the beginning of the file and see the instructions.

**Comment by bsekachev** [comment]
@klakhov 
Do you have some thougs about the PR?

**Comment by klakhov** [comment]
@bsekachev 
Patch looks ok, but is there a way we can test it? There is chance that other input fields are affected

**Comment by bsekachev** [comment]
I tested it manually, and we have tests for attributes as far as I remember.
All the input fields are affected for the unification, I do not see the problem here.
Unfortunately I can't test with Chinese language

**Comment by klakhov** [comment]
@bsekachev
Ive tried it with windows chineese and it seems with patch its working well. We should merge it
Animation1
Animation2

**Review by bsekachev — APPROVED** [review]

**Pull request merged**